starter-code/src/functions-and-arrays.js

```diff
@@ -74,6 +181,21 @@ var wordsCount = [
   'disobedience',
   'matter'
 ];
+
+function howManyTimes (arrayWords, searchWord){
+  if (arrayWords.length === 0){
+    return false;
+  }
+  var wordCount = 0;
+  for (var i = 0; i < arrayWords.length; i++){
+    if (searchWord === arrayWords[i]){
+      wordCount += 1;
+    }
+  }
+  return wordCount;
```

Comment:
Es correcto, con un filter( ) os hubiese quedado muy cortito. Cada vez que veáis algo como encontrar algo o varios algos en una lista siguiendo un patrón, os recomiendo pensar primero en filter( )